src/PushChannelSubscriptions.php

```diff
@@ -30,4 +31,16 @@ public function save ( $subscription ) {
         return new PushChannelSubscription ( $body );
     }
 
+    /**
+     *  Returns a PaginatedResult object with the list of PushChannelSubscription
+     *  objects, filtered by the given parameters.
+     *
+     *  @param array $params the parameters used to filter the list
+     */
+    public function list_ (array $params = []) {
```

Comment:
@jdavid if you are using a method that is different to the spec, then you need to update the docs in https://github.com/ably/docs.  Can you do a PR for that to update `general/push/admin`?

@tomczoink see below.  Do we need to create an issue for this?

![screenshot_2019-05-31_03-27-07_pm](https://user-images.githubusercontent.com/43789/58712583-c3942d80-83b8-11e9-9a56-c999818977c7.png)

Reply:
In fact @tomczoink, the problem is worse.  Where has Push gone in the documentation nav??  See below for the docs repo nav.

![screenshot_2019-05-31_03-29-25_pm](https://user-images.githubusercontent.com/43789/58712669-eb839100-83b8-11e9-8363-456f2b47b339.png)